pubspec.lock

```diff
@@ -0,0 +1,992 @@
+# Generated by pub
```

Comment:
Please remove this lock file from commit, but keep this in your local.
See Why: https://dart.dev/guides/libraries/private-files#pubspeclock